docs/admin/known_issues.rst

```diff
@@ -22,13 +22,27 @@ Current known issues
 - Updates are slow due to the number of VMs involved, and due to some updates
   being fetched over Tor. We are investigating various strategies to improve
   performance and reliability of the updater.
+- SecureDrop instances with very large numbers of sources may encounter
+  UI performance issues. While performance improvements are on the roadmap,
+  `our recommendation <https://docs.securedrop.org/en/stable/admin/maintenance/backup_and_restore.html#minimizing-disk-use>`_ 
+  is to delete information from the servers as regularly as possible, both
+  for performance and security reasons.
+- There is currently no mechanism for cancelling or retrying file downloads.
+  This feature is planned.
 - Currently, only LUKS-encrypted *Export Devices* are supported. VeraCrypt support
   will be added in a future release.
 - Printer support is limited to a specific HP printer model, and printing
   different file types is not as reliable yet as under Tails. Support for
   additional non-networked printers will be added in a future release.
 - Currently, only app-based two-factor authentication (TOTP) is supported.
-- The SecureDrop Client can only be configured with a single *Submission Key*.
-  If the *Submission Key* for your SecureDrop server was rotated in the past,
-  and submissions encrypted with the old key are still on your server, you will
-  not be able to open them with the SecureDrop Client.
+- The SecureDrop Client does not currently handle files that are "double-encrypted"
+  (when a source pre-encrypts submission locally before uploading them to SecureDrop).
+  Until this is fully supported, we suggest using the Tails-based Secure Viewing
+  Station setup for pre-encrypted submissions.
```

Comment:
```suggestion
  Until this is fully supported, we suggest using the Tails-based 
  *Secure Viewing Station* for pre-encrypted submissions.
```

Secure Viewing Station needs to be italicized because it's a glossary term. I also suggest removing "setup", as it's not necessary.